lib/active_model/serializer.rb

```diff
@@ -4,6 +4,7 @@
 require 'active_model/serializer/config'
 
 require 'thread'
+require 'concurrent/map'
```

Comment:
concurrent is not an dependency of this project in this branch, so we need to add it as a dependency.